portability/toku_time.h

```diff
@@ -98,9 +98,17 @@ double tokutime_to_seconds(tokutime_t)  __attribute__((__visibility__("default")
 
 // Get the value of tokutime for right now.  We want this to be fast, so we expose the implementation as RDTSC.
 static inline tokutime_t toku_time_now(void) {
+#if defined(x86_64) || defined(__i386__)
```

Comment:
Fails to build on all supported intel platforms (all 64 bit), this should probably be __x86_64__

```22:36:07 /mnt/workspace/PerconaFT-param/BUILD_TYPE/release/Host/ubuntu-precise-64bit/portability/toku_time.h:110:2: error: #error No timer implementation for this platform
22:36:07  #error No timer implementation for this platform
22:36:07   ^
22:36:07 /mnt/workspace/PerconaFT-param/BUILD_TYPE/release/Host/ubuntu-precise-64bit/portability/toku_time.h: In function ‘tokutime_t toku_time_now()’:
22:36:07 /mnt/workspace/PerconaFT-param/BUILD_TYPE/release/Host/ubuntu-precise-64bit/portability/toku_time.h:112:1: error: no return statement in function returning non-void [-Werror=return-type]
22:36:07  }
22:36:07  ^
22:36:07 cc1plus: all warnings being treated as errors```

see http://jenkins.percona.com/view/TokuDB/job/PerconaFT-param/54/